tools/coqdoc/output.ml

```diff
@@ -655,10 +656,23 @@ module Html = struct
 
   let reference s r =
     match r with
-    | Def (fullid,ty) ->
+    | Def [fullid,ty] ->
         let s' = sanitize_name fullid in
         printf "<a id=\"%s\" class=\"idref\" href=\"#%s\">" s' s';
         printf "<span class=\"id\" title=\"%s\">%s</span></a>" (type_name ty) s
+    | Def l ->
+        let ty =
+          l |> List.map (fun (_,ty) -> type_name ty)
+          |> CList.sort_uniquize String.compare
+          |> String.concat "," in
```

Comment:
The `title` attribute is meant to be read by humans. You should use `", "` (extra space).